modules/light-clients/08-wasm/test_data/README

```diff
@@ -0,0 +1,59 @@
+How to regenerate grandpa test data
```

Comment:
tysm for the README, haven't tried to run these atm but really great idea you decided to add this

Reply:
+1 thanks for this doc